tests/foreman/cli/test_discoveryrule.py

```diff
@@ -582,16 +640,20 @@ def test_positive_create_rule_with_non_admin_user(self):
         :CaseLevel: Integration
         """
         rule_name = gen_string('alpha')
+        options = {
+            'name': rule_name,
+            'search': 'cpu_count = 5',
+            'organization-ids': self.org['id'],
+            'location-ids': self.loc['id'],
```

Comment:
can't you use names right away?

Reply:
should i?

Reply:
theoretically tomorrow names can be removed from command instead of ids, i'm not sure what's the best approach here

Reply:
what is difference? why do you need to use both ids and names? why do you need to check for bz here? why not? why shouldn't you?

Reply:
ok, then we will fix it tomorrow